atoma-service/src/handlers/chat_completions.rs

```diff
@@ -1062,7 +1108,7 @@ pub mod utils {
                 .map_err(|e| {
                     AtomaServiceError::InternalError {
                         message: format!(
-                            "Error sending encryption request, for request with payload hash: {:?}, and stack small id: {}, with error: {}",
+                            "Error sending encryption request, for request with payload hash: {:?}, and stack small id: {:?}, with error: {}",
```

Comment:
why ?

Reply:
the stack_id is optional, I agree I should change to either stack or user_address, because one is being filled.